packages/components/pie-select/src/index.ts

```diff
@@ -86,7 +86,6 @@ export class PieSelect extends FormControlMixin(RtlMixin(PieElement)) implements
         this._internals.setFormValue(this._select.value);
     }
 
-    @property()
```

Comment:
Could you try moving this to the `set` instead? Then we may also no longer need to call `requestUpdate`